### DISCUSSION TITLE
Would updating the cask of SensibleSideButtons to fork be allowed?

### DISCUSSION THREAD

**Discussion by nphmuller** [Homebrew/discussions#3655] [Casks]
The original project hasn't been updated for years:

https://sensible-side-buttons.archagon.net
https://github.com/archagon/sensible-side-buttons

There seems to be a fork that's slowly growing in popularity, with added features like native Apple Silicon support:
https://github.com/thealpa/SaneSideButtons
https://janhuelsmann.com/sanesidebuttons

If I were to submit a pull request to move the cask to this forked version, is there a good chance this pull request would be allowed? Or is it against certain rules?

Thanks for the help!

**Comment by nphmuller** [comment]
Hmm... Now that I look at it better, the fork has been renamed to SaneSideButtons. So I guess submitting a new cask would be the best way to go here, right?

**Reply by carlocab** [reply]
A new cask makes sense.
